src/message.rs

```diff
@@ -760,6 +763,10 @@ impl Message {
             .await
             .is_ok()
     }
+
+    pub fn get_error(&self) -> Option<String> {
```

Comment:
Is there precedent for us providing this kind of API?  IIRC the rusty thing to do is call this the same as the field: `error()`.